src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainParameters.cs

```diff
@@ -149,5 +149,10 @@ public class ChainParameters
     /// <see href="https://eips.ethereum.org/EIPS/eip-4844#parameters">EIP-4844</see>.
     /// </summary>
     public ulong? Eip4844TargetBlobGasPerBlock { get; set; }
+
+    /// <summary>
+    /// Enables blob gas fee collection for Gnosis chain
+    /// </summary>
+    public ulong? Eip4844PectraTransitionTimestamp { get; set; }
```

Comment:
`Eip4844FeeCollectorTransition`

Reply:
For me this name is less clear since there is no `Eip4844FeeCollector`, we're just using one `FeeCollector` for everything - the new name would imply there is a separate fee collector for 4844. Also if we consistently use the name `Eip4844Pectra` everywhere it's easier to search for.

Reply:
yes but we are transitioning 4844 fees

Reply:
It's a bit long but how about `Eip4844PectraFeeCollectionTransitionTimestamp`

Reply:
We don't call EIP's by fork name.

Reply:
So this is name from the spec? Then ok, but sounds meh.

Reply:
I agree using `Eip4844FeeCollector` everywhere is a lot clearer though, happy to do that